pkg/frontend/adminactions/mustgather.go

```diff
@@ -151,24 +141,27 @@ func (ka *kubeactions) MustGather(ctx context.Context, oc *api.OpenShiftCluster,
 		return err
 	}
 
-	ka.log.Info("waiting for must-gather pod")
-	err = ka.waitForPodRunning(ctx, cli, pod)
+	a.log.Info("waiting for must-gather pod")
+	err = a.waitForPodRunning(ctx, a.k8sClient, pod)
 	if err != nil {
 		return err
 	}
 
-	ka.log.Info("must-gather pod running")
-	rc, err := portforward.ExecStdout(ctx, ka.log, ka.env, oc, pod.Namespace, pod.Name, "copy", []string{"tar", "cz", "/must-gather"})
+	a.log.Info("must-gather pod running")
+	rc, err := portforward.ExecStdout(ctx, a.log, a.env, a.oc, pod.Namespace, pod.Name, "copy", []string{"tar", "cz", "/must-gather"})
 	if err != nil {
 		return err
 	}
 	defer rc.Close()
 
+	w.Header().Add("Content-Type", "application/gzip")
+	w.Header().Add("Content-Disposition", `attachment; filename="must-gather.tgz"`)
```

Comment:
Moved here to be consistent with https://github.com/Azure/ARO-RP/pull/886#discussion_r463724754